packages/dynamic-import/client.js

```diff
@@ -1,22 +1,44 @@
 var Module = module.constructor;
 var cache = require("./cache.js");
+var Meteor = require("meteor/meteor").Meteor;
 var meteorInstall = require("meteor/modules").meteorInstall;
 var dynamicVersions = require("./dynamic-versions.js");
 
+// Fix for Safari bug https://bugs.webkit.org/show_bug.cgi?id=226547
+function idbReady() {
+  var isSafari = !navigator.userAgentData &&
+    /Safari\//.test(navigator.userAgent) &&
+    !/Chrom(e|ium)\//.test(navigator.userAgent);
+  // No point putting other browsers through this mess.
+  if (!isSafari || !window.indexedDB.databases) return Promise.resolve();
```

Comment:
It's probably safer to check if indexedDB also exists:
!window.indexedDB || !window.indexedDB.databases

Reply:
Fixed!